website/docs/d/datasource_google_service_account_id_token.html.markdown

```diff
@@ -0,0 +1,128 @@
+---
+subcategory: "Cloud Platform"
+layout: "google"
+page_title: "Google: google_service_account_id_token"
+sidebar_current: "docs-google-service-account-id-token"
+description: |-
+  Produces OpenID Connect token for service accounts
+---
+
+# google\_service\_account\id\_token
+
+This data source provides a google OpenID Connect (`oidc`) `id_token` 
+
+For more information see
+[OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html#IDToken) as well as [Authenticating using Google OpenID Connect Tokens](https://github.com/salrashid123/google_id_token)
```

Comment:
Is there not an official GCP docs site that explains this?

Reply:
unfortunately, no, thats the only doc about google OIDC tokens...

we do have:
  https://developers.google.com/identity/protocols/oauth2/openid-connect
but thats just focued on oidc tokens for user flow (whcih isn't used here as such)

as an internal ref: see b/136495350